### ISSUE TITLE
Retraining models required for fixing #593

### ISSUE THREAD

**Issue by schen149** [CogComp/cogcomp-nlp#693]
TL;DR: We will need to retrain PrepSRL, Quantifier, (maybe) Question Typer

The [Fix](https://github.com/CogComp/cogcomp-nlp/commit/0db5962ba071db7d8438aa958abf411e732c2218#diff-2eacf31fd2ccc45f721ab67ab2db70acR62) applied to #593 (currently residing in branch [string_join_require_retrain](https://github.com/CogComp/cogcomp-nlp/tree/string_join_require_retrain)) will affect the behavior of [FeatureNGramUtility ](https://github.com/CogComp/cogcomp-nlp/blob/7d9dad3fedc16ac59feb278815e27dc195d1367e/edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/helpers/FeatureNGramUtility.java) in edison. All 4 feature generating functions in FeatureNGramUtility calls StringUtils.join() to produce feature names, which currently have trailing dashes. The fix removes the trailing dash, but also changes the feature names. 

Here are all packages that will (likely) be affected --

- PrepSRL
https://github.com/CogComp/cogcomp-nlp/blob/7d9dad3fedc16ac59feb278815e27dc195d1367e/prepsrl/src/main/java/edu/illinois/cs/cogcomp/prepsrl/features/WordBigrams.java#L38-L39

- Quantifier
https://github.com/CogComp/cogcomp-nlp/blob/7d9dad3fedc16ac59feb278815e27dc195d1367e/quantifier/src/main/java/edu/illinois/cs/cogcomp/quant/features/WordBigrams.java#L44-L45

- Question Typer 
(it looks like just a debug/helper function, not actually used in model? but not sure)
https://github.com/CogComp/cogcomp-nlp/blob/7d9dad3fedc16ac59feb278815e27dc195d1367e/question-type/src/main/java/edu/illinois/cs/cogcomp/question_typer/QuestionTyperFeatureExtractorsUtils.java#L191-L200
